spec.bs

```diff
@@ -347,6 +530,196 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
     1. Return the value of [=addModule success=].
   </div>
 
+  ## {{SharedStorageUrlWithMetadata}} and Reporting ## {#reporting}
+
+  A {{SharedStorageUrlWithMetadata}} {{/object}} is a [=dictionary=] containing a [=string=] representing a [=/URL=] and, optionally, a {{reportingMetadata}} {{/object}}.
+
+  <xmp class='idl'>
+    dictionary SharedStorageUrlWithMetadata {
+      required USVString url;
+      object reportingMetadata;
```

Comment:
Not that I'm aware of. cc-ing @pythagoraskitty  who authored the change.

Reply:
@pythagoraskitty could you take a look? i.e. Is there a reason `reportingMetadata` is an `object` instead of [any](https://webidl.spec.whatwg.org/#idl-any)?

Reply:
Well, `reportingMetadata` needs to be a dictionary, so the Chromium implementation currently assumes it's an `object`: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/shared_storage/shared_storage_worklet.cc;drc=4ec42762cc8329e274218f13828754b17941c2ff;l=3

Reply:
If reporting metadata is supposed to be a dictionary, then it should have its own dictionary type (just like `SharedStorageUrlWithMetadata`) and just be embedded inside `SharedStorageUrlWithMetadata` as a member.

Reply:
I think it can be a dictionary / don't see any reason why it cannot be. Created issue https://github.com/WICG/shared-storage/issues/141 to track, so that @pythagoraskitty can respond and/or address this separately.